src/notation/view/widgets/selectdialog.ui

```diff
@@ -34,47 +34,46 @@
         <string>Select</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
-        <item row="1" column="0">
+        <item row="2" column="0">
          <widget class="QCheckBox" name="sameSubtype">
           <property name="text">
            <string>Same subtype:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
-         <widget class="QLineEdit" name="type">
-          <property name="readOnly">
-           <bool>true</bool>
+         <widget class="QLabel" name="type"/>
```

Comment:
I'm afraid the change to QLabel may mean that it becomes inaccessible for screen reader users. Perhaps setting the "focus policy" may mitigate that?

Reply:
I removed the `tabstop` tag below, would adding that back in be enough?

Reply:
I'm afraid it won't, because labels are by default excluded from keyboard navigation anyway. However I must say I don't really know this for sure, so the best way to find out is to give it a try. 
(If you don't have time to set up a screen reader, then I think it's enough to verify that the label can be somehow accessed using the keyboard.)